src/Compiler/Driver/CompilerConfig.fs

```diff
@@ -47,6 +47,9 @@ let FSharpScriptFileSuffixes = [ ".fsscript"; ".fsx" ]
 let FSharpIndentationAwareSyntaxFileSuffixes =
     [ ".fs"; ".fsscript"; ".fsx"; ".fsi" ]
 
+let FsharpExperimentalFeaturesEnabledAutomatically =
+    Environment.GetEnvironmentVariable("FSHARP_EXPERIMENTAL_FEATURES") |> isNotNull
```

Comment:
It would probably be better to make it possible to control different experimental features and to not reuse one flag?